README.md

```diff
@@ -13,121 +13,274 @@ and the Flutter guide for
+
+## 📖 Additional Information
 
 The source code can be accessed via [GitHub repository](https://github.com/anusii/solid_auth). You can also file issues you face at [GitHub Issues](https://github.com/anusii/solid_auth/issues).
 
-### Running Solid Auth in web applications
-
-In order to successfully run `solid auth` in a web application you also need to create a custom `callback.html` file inside the `web` directory. After created simply copy and paste the following code into that file.
-
-```html
-<!DOCTYPE html>
-<html>
-
-<head>
-    <script>
-        const AUTH_DESTINATION_KEY = "openidconnect_auth_destination_url";
-        const AUTH_RESPONSE_KEY = "openidconnect_auth_response_info";
-
-        window.onload = function () {
-            if (window.opener && window.opener !== window) { //Used when working as a popup. Uses post message to respond to the parent window                
-                var parent = window.opener ?? window.parent;
-                parent.postMessage(location.href, "*");
-            } else { //Used for redirect loop functionality.                
-                //Get the original page destination                
-                const destination = sessionStorage.getItem(AUTH_DESTINATION_KEY || "/");
-                sessionStorage.removeItem(AUTH_DESTINATION_KEY);
-                //Store the current window location that will be used to get the information for authentication
-                sessionStorage.setItem(AUTH_RESPONSE_KEY, window.location);
-
-                //Redirect to where we're going so that we can restore state completely
-                location.assign(destination);
-            }
-        }
-    </script>
-</head>
-
-<body>
-</body>
-
-</html>
-```
+An example project that demonstrates `solid_auth` usage can be found [here](https://github.com/anusii/solid_auth/tree/main/example).
+
+## 🙏 Acknowledgments
+
+This library builds upon the excellent work of the [Bdaya-Dev/oidc](https://github.com/Bdaya-Dev/oidc) team. We are standing on the shoulders of giants! 
+
+Special thanks to:
+- **[Bdaya-Dev/oidc](https://pub.dev/packages/oidc)** - The robust, well-maintained OpenID Connect implementation that powers this library
+- **[oidc_default_store](https://pub.dev/packages/oidc_default_store)** - Secure, platform-appropriate token storage
+- The broader Solid and OpenID Connect communities for their specifications and guidance
+
+The solid_auth library focuses specifically on Solid pod authentication while leveraging these excellent foundational libraries for the core OIDC functionality.
+
+## 🔗 Links
+
+- [Solid Project](https://solidproject.org/)
+- [Solid OIDC Specification](https://solid.github.io/solid-oidc/)  
+- [WebID Specification](https://www.w3.org/2005/Incubator/webid/spec/identity/)
+- [Example Application](https://github.com/anusii/solid_auth/tree/main/example)
+- [Issue Tracker](https://github.com/anusii/solid_auth/issues)
+
+---
+
+## Roadmap
+
```

Comment:
And do you think we should also discuss here that the solid specifications for authorizing applications are still in the works and once authorization flow like in  https://solid.github.io/data-interoperability-panel/specification/ is available in Solid Server implementations, we want to support it? Currently usually the pod servers give an authenticated application (like our example) full access to the entire pod - I think that it would be valuable for users of solid_auth to be aware of this and that it is a problem in the solid ecosystem on which people are working.